### PR TITLE
Claw

### PR DESCRIPTION
**Pull request by rohanmishr**
# Purpose
Add claw code

# Changes
- RobotContainer.java: Added claw, changed button bindings
- AdjustArm.java: Fixed joystick inversions
- MoveArm.java: Changed presets to updated and revised values
- MoveClaw.java: Added claw movement code
- Arm.java: Added tuned PID values and calibration
- Claw.java: Added claw code

### CODE REVIEW

**Review by bjmcternan — DISMISSED** [review]
This looks good to me.

**Review by GoldCrafter — APPROVED** [review]
Looks good

**Review by bjmcternan — CHANGES_REQUESTED** [review]
Quick naming change

File: src/main/java/frc/robot/commands/MoveArm.java [hunk 1]
```diff
@@ -46,20 +46,16 @@ public void initialize() {
     public void execute() {
         switch(m_node) {
             case HYBRID:
-                m_arm.rotateTo(SCORE_HYBRID_ANGLE);
-                m_arm.extendTo(SCORE_HYBRID_X_INCHES);
+                m_arm.setPosition(SCORE_HYBRID_ANGLE, SCORE_HYBRID_X_INCHES);
```
[the remark this answers is not in the document]
Comment (GoldCrafter) [reply src/main/java/frc/robot/commands/MoveArm.java]: Changed X to R

**Review by GoldCrafter — APPROVED** [review]
Looks good

**Review by bjmcternan — APPROVED** [review]
Looks good

**Pull request merged**